beast/tools/trim_many_via_obsdata.py

```diff
@@ -60,6 +60,11 @@
         sed_trimname = filebase + "_seds_trim.grid.hd5"
         noisemodel_trimname = filebase + "_noisemodel_trim.grid.hd5"
 
+        # if these already exist, then continue to the next set of files to trim
+        if os.path.isfile(sed_trimname) and os.path.isfile(noisemodel_trimname):
+            print("trimming already complete for "+sed_trimname)
+            continue
+
```

Comment:
This is not related to the main topic of this PR.  Could be a separate PR.

Not sure that not just automatically overwriting the trim files isn't a better solution.  That is what would currently happen, right?  Or is the current behavior to crash?

Reply:
Happy to move it to a new PR.

The issue is that trimming can take a while, so skipping the file makes it run faster.

Reply:
Please do.  I mainly want to make sure that this change does not mess up runs as right now I'm assuming that is does replace the file.  Would be a bummer if this silently changed to *not* updating the file and this was not the desired behavior.

Reply:
Opened a new PR for this (#501).